PurchasesCoreSwift/Logging/Strings/Strings.swift

```diff
@@ -11,8 +11,8 @@ import Foundation
     static let identity = IdentityStrings()
     static let network = NetworkStrings()
     static let offering = OfferingStrings()
-    @objc public static let purchase = PurchaseStrings()
+    static let purchase = PurchaseStrings()
```

Comment:
Based on @Juanpe's suggestion, we then wouldn't need to do all this initialization either...